meta-printnanny/recipes-core/printnanny-gst-plugin/printnanny-gst-plugin_0.4.0.bb

```diff
@@ -4,8 +4,22 @@ HOMEPAGE = "https://github.com/bitsy-ai/printnanny-gst-plugin-rs"
 SRC_URI = "\
     https://github.com/bitsy-ai/printnanny-gst-plugin-rs/releases/download/printnanny-gst-plugin-v${PV}/printnanny-gst-plugin-v${PV}-${TARGET_ARCH}-unknown-linux-gnu.tar.gz \
 "
+
+SRC_URI[sha256sum] = "${@bb.utils.contains( \
+    'TARGET_ARCH', \
+    'aarch64', \
+    '78b8b9f15800aa5eeb75ad3e69c519b66cdd3e7ecbae910f69c2442b99fd3d9c', \
+    bb.utils.contains( \
+      'TARGET_ARCH', \
+      'x86_64', \
+      '3439fd013133a3c58440666a61338a03ffdaced819c7e53257e4730902b2c0bd', \
+      '', \
+      d), \
+    d)}"
```

Comment:
```suggestion
SRC_URI:aarch64 = "\
    https://github.com/bitsy-ai/printnanny-gst-plugin-rs/releases/download/printnanny-gst-plugin-v${PV}/printnanny-gst-plugin-v${PV}-${TARGET_ARCH}-unknown-linux-gnu.tar.gz;sha256sum=78b8b9f15800aa5eeb75ad3e69c519b66cdd3e7ecbae910f69c2442b99fd3d9c \
"
SRC_URI:x86-64 = "\
    https://github.com/bitsy-ai/printnanny-gst-plugin-rs/releases/download/printnanny-gst-plugin-v${PV}/printnanny-gst-plugin-v${PV}-${TARGET_ARCH}-unknown-linux-gnu.tar.gz;sha256sum=3439fd013133a3c58440666a61338a03ffdaced819c7e53257e4730902b2c0bd \
"
```

Reply:
ah, great. that didn't come up when I was researching how to combine variable flags and overrides.